cpp/tests/utilities/column_utilities.cu

```diff
@@ -352,7 +352,11 @@ struct column_view_printer {
     if( col.size()==0 )
       return;
     std::vector<std::string> keys = to_strings(dictionary.keys());
-    std::vector<std::string> indices = to_strings(dictionary.indices());
+    std::vector<std::string> indices = to_strings(
```

Comment:
Same comment as above.  Seems like if you're passing in an offset, you should be sending in ::head(), or vice-versa.

Reply:
The indices child column will never have offsets so `data()` and `head()` should always be the same.